larray/__init__.py

```diff
@@ -8,7 +8,7 @@
 from larray.core.array import (LArray, zeros, zeros_like, ones, ones_like, empty, empty_like, full,
                                full_like, sequence, labels_array, ndtest, aslarray, identity, diag,
                                eye, all, any, sum, prod, cumsum, cumprod, min, max, mean, ptp, var,
-                               std, median, percentile, stack)
+                               std, median, percentile, stack, zip_array_values, zip_array_items)
```

Comment:
I think these 2 functions should be added in api.rst
Even if the users don't use them by themselves, they may want to be able to read the documentation and see the examples after *we* add them to their code.

Reply:
ok